src/containers/TermsOfUse/index.tsx

```diff
@@ -0,0 +1,330 @@
+import React, {FC} from 'react';
+
+import {A, PageTitle} from 'components';
+
+import './TermsOfUse.scss';
+
+const TermsOfUse: FC = () => {
+  return (
+    <>
+      <PageTitle title="Terms Of Use" />
+      <div className="TermsOfUse">
+        <p>Effective as of April 5, 2021.</p>
+
+        <h1 className="TermsOfUse__title">Terms of Use</h1>
+        <p>
+          Welcome to thenewboston (the "Service"). The following Terms of Use apply when you view or use the Service
+          located at:{' '}
+          <A href="https://thenewboston.com" newWindow={false}>
+            https://thenewboston.com
+          </A>
+          . Please review the following terms carefully. By accessing or using the Service, you signify your agreement
+          to these Terms of Use. If you do not agree to these Terms of Use, you may not access or use the Service.
+        </p>
+
+        <h2 className="TermsOfUse__subtitle">Privacy Policy</h2>
+        <p>
+          The company respects the privacy of its Service users. Please refer to the Company's Privacy Policy which
+          explains how we collect, use, and disclose information that pertains to your privacy. When you access or use
+          the Service, you signify your agreement to this Privacy Policy.
+        </p>
+        <h2 className="TermsOfUse__subtitle">Registration; Rules for User Conduct and Use of the Service</h2>
+        <p>
+          You need to be at least 16 years old to register for and use the Service. If you are a user who signs up for
+          the Service, the company will create a personalized account, which includes a unique username and a password
+          to access the Service and allow you to receive messages from the Company. You agree to notify us immediately
+          of any unauthorized use of your password and/or account. The Company will not be responsible for any
+          liabilities, losses, or damages arising out of the unauthorized use of your member name, password and/or
+          account.
+        </p>
+
+        <h2 className="TermsOfUse__subtitle">Use Restrictions</h2>
+        <p>
+          Your permission to use the Site is conditioned upon the following Use Restrictions and Conduct Restrictions:
```

Comment:
Site => Website 

And I think this one has to either be defined above here, or a link to the other file that has some definition.